AudioAnalytics/AudioAnalyzer/src/main/java/sample/MediaServicesEventProcessor.java

```diff
@@ -42,75 +65,55 @@ public MediaServicesEventProcessor() {
         this.JOB_NAME = null;
         this.MONITOR = null;
         this.LIVE_EVENT_NAME = null;
+        this.EVENT_HUB_NAME = null;
+        this.EVENT_HUB_CONN_STRING = null;
+        this.BLOB_CONTAINER = null;
     }
 
-    /**
-     * OnOpen is called when a new event processor instance is created by the host. 
-    */
-    @Override
-    public void onOpen(PartitionContext context) throws Exception
-    {
-        System.out.println("Partition " + context.getPartitionId() + " is opening");
-    }
-
-    /** 
-     * OnClose is called when an event processor instance is being shut down.
-    */
-    @Override
-    public void onClose(PartitionContext context, CloseReason reason) throws Exception
-    {
-        System.out.println("Partition " + context.getPartitionId() + " is closing for reason " + reason.toString());
-    }
-
-    /**
-     * onError is called when an error occurs in EventProcessorHost code that is tied to this partition, such as a receiver failure.
-     */
-    @Override
-    public void onError(PartitionContext context, Throwable error)
-    {
-        System.out.println("Partition " + context.getPartitionId() + " onError: " + error.toString());
-    }
-    
-    /**
-     * onEvents is called when events are received on this partition of the Event Hub. 
-     */
-    @Override
-    public void onEvents(PartitionContext context, Iterable<EventData> events) throws Exception
-    {
-        for (EventData eventData: events) {
-            printEvent(context, eventData);
+    public void stop() {
+        if (this.MONITOR instanceof EventProcessorClient) {
```

Comment:
Is it possible for it not to be EventProcessorClient?

Reply:
According to your comments, it has been fixed in the new version.